Makefile

```diff
@@ -13,9 +13,10 @@ rundep=go run -modfile misc/devdeps/go.mod
 # NOTE: these are not very useful in this makefile, but they serve as
 # documentation for sub-makefiles.
 
-# disable cgo by default. cgo requires some additional dependencies in some
-# cases, and is not strictly required by any tm2 code.
-CGO_ENABLED ?= 0
+# Enable cgo by default. cgo is needed to enable HID enumeration and thus ledger support.
+# However cgo requires some additional dependencies in some
+# cases, and is not required by any tm2 code so disable if you don't need ledger support.
+CGO_ENABLED ?= 1
```

Comment:
Why is it in the main Makefile and not just in `gno.land/cmd/gnokey`?

Reply:
I think it's because we need the it in an env variable during the build process, the underlying zondax lib expects it (there is no Go param or anything similar)